copter/source/docs/traditional-helicopter-autotune.rst

```diff
@@ -0,0 +1,199 @@
+|                             | +---------+----------------------+ |
+|                             | | 7       | Feedforward, Rate D*,| |
+|                             | |         | Rate P, & Angle P    | |
+|                             | +---------+----------------------+ |
+|                             | | 8       | Max Gain             | |
+|                             | +---------+----------------------+ |
++-----------------------------+------------------------------------+
+
+Max gain determination is done before tuning Rate D and Rate P gains.
+
+:ref:`Axes Bitmask<AUTOTUNE_AXES__AC_AutoTune_Heli>`
+-----------------------------------------------------------
+
+Specifies one or more axes to be tuned.
+
++----------------------+---------------------------------+
+| Bits                 | Values                          |
++======================+=================================+
+| +-------+----------+ | +---------+-------------------+ |
+| | Bits  | Axis     | | | Values  | Axes tested       | |
+| +=======+==========+ | +=========+===================+ |
+| | 0     | Roll     | | | 1       | Roll              | |
+| +-------+----------+ | +---------+-------------------+ |
+| | 1     | Pitch    | | | 2       | Pitch             | |
+| +-------+----------+ | +---------+-------------------+ |
+| | 2     | Yaw      | | | 3       | Roll & Pitch      | |
+| +-------+----------+ | +---------+-------------------+ |
+|                      | | 4       | Yaw               | |
+|                      | +---------+-------------------+ |
+|                      | | 5       | Roll & Yaw        | |
+|                      | +---------+-------------------+ |
+|                      | | 6       | Pitch & Yaw       | |
+|                      | +---------+-------------------+ |
+|                      | | 7       | Roll, Pitch & Yaw | |
+|                      | +---------+-------------------+ |
++----------------------+---------------------------------+
+
+:ref:`Maximum Response Gain<AUTOTUNE_GN_MAX>`
+---------------------------------------------
+
+Specifies the maximum response gain to be used to tune the Rate D, Rate P and Angle P gains.  The response gain is the output of the response divided by the input.  It is recommended that 1.8 to 2.0 be used for pitch and roll axes as these axes are typically more lightly damped.  It is recommended that 1.0 to 1.4 be used for the yaw axis.
+
+
+:ref:`Minimum Test Frequency<AUTOTUNE_FRQ_MIN>`
+-----------------------------------------------
+
+Specifies the minimum frequency in radians per second used during the dwell or frequency sweeps.  For frequency sweeps, this will be the starting frequency.
+
+
+:ref:`Maximum Test Frequency<AUTOTUNE_FRQ_MAX>`
+-----------------------------------------------
+
+Specifies the maximum frequency in radians per second used during the dwell or frequency sweeps.  For frequency sweeps, this will be the ending frequency.
```

Comment:
same comment as above

Reply:
Again I have added some words to address your comment but this one is even harder to tell users how to adjust